test/e2e/cluster_upgrade.go

```diff
@@ -30,6 +30,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/ginkgowrapper"
 	"k8s.io/kubernetes/test/e2e/upgrades"
+	apps "k8s.io/kubernetes/test/e2e/upgrades/apps"
```

Comment:
upgrades/apps should be imported as apps. I think this is a holdover from when it was upgrades/sigapps and you wanted to import it as apps.